schemas/context/profile.schema.json

```diff
@@ -86,6 +91,35 @@
           "description":
             "The unit ID within the organization owning the profile. This ID can be used to reference the organization details maintained in another dataset."
         },
+        "xdm:preferredLanguage": {
+          "title": "Preferred Language",
+          "type": "string",
+          "pattern":
+            "^(((([A-Za-z]{2,3}(-([A-Za-z]{3}(-[A-Za-z]{3}){0,2}))?)|[A-Za-z]{4}|[A-Za-z]{5,8})(-([A-Za-z]{4}))?(-([A-Za-z]{2}|[0-9]{3}))?(-([A-Za-z0-9]{5,8}|[0-9][A-Za-z0-9]{3}))*(-([0-9A-WY-Za-wy-z](-[A-Za-z0-9]{2,8})+))*(-(x(-[A-Za-z0-9]{1,8})+))?)|(x(-[A-Za-z0-9]{1,8})+)|((en-GB-oed|i-ami|i-bnn|i-default|i-enochian|i-hak|i-klingon|i-lux|i-mingo|i-navajo|i-pwn|i-tao|i-tay|i-tsu|sgn-BE-FR|sgn-BE-NL|sgn-CH-DE)|(art-lojban|cel-gaulish|no-bok|no-nyn|zh-guoyu|zh-hakka|zh-min|zh-min-nan|zh-xiang)))$",
+          "description":
+            "Describes the preferred system of communication used by the profile. Language codes are expressed in BCP 47 format, for example `en-GB`."
+        },
+        "xdm:timeZone": {
+          "title": "Time Zone",
+          "type": "string",
+          "description":
+            "Describes which time zone the profile is present in, most frequently/the time zone preferred by the profile."
```

Comment:
I missed this before. What's the format for time zones? If we don't have a defined format, I'd suggest using the IETF `tz` database: https://www.ietf.org/timezones/tzdb-2016i/tz-link.htm

We can probably build a `pattern` regex and *should* provide a couple of `examples`.